awswrangler/s3/_write_parquet.py

```diff
@@ -224,54 +211,6 @@ def _to_parquet(
     return paths
 
 
-def _to_parquet_distributed(  # pylint: disable=unused-argument
```

Comment:
Thanks for moving these out into a separate package! Less conditional imports, no more mixed distributed and non-distributed code. Definitely a highlight of this PR!